discord.py 2 and Slash command upgrade

reddit-tennis opened the pull request:
Upgrade to Discord py2 https://discordpy.readthedocs.io/en/stable/migrating.html - will now work in threads etc. 

And converted to slash commands. However, not all commands are converted to slash commands as some don't work 👀 

Examples:

![image](https://user-images.githubusercontent.com/38725593/234122302-9a4d5c44-d54c-4bc4-a742-f5cd05ee1ced.png)

![image](https://user-images.githubusercontent.com/38725593/234122382-4c658c80-44ae-43aa-8149-9ac53bdedda1.png)

ingenium21 commented:
CORRRRR

AndyReifman commented:
What needs to be done to complete this PR? It looks like it might be in a state to merge what we've got to allow others to contribute to bringing the entirety of the codebase up to date.

reddit-tennis commented:
> What needs to be done to complete this PR? It looks like it might be in a state to merge what we've got to allow others to contribute to bringing the entirety of the codebase up to date.

Good idea. Will go through them all later and let you know.

More of a demo to others to see their thoughts on slash commands

reddit-tennis commented:
All good to review finally. 

I have changed some commands back to normal (europa, copa, olympic, anysignings, copy, wengersucks)

Note I changed the way the token is loaded, simpler this way.

ingenium21 commented:
> 

I think those probably didn't work because they aren't on right now and fotmob doesn't return anything for them.  Not anything wrong with slash commands. 

Either way, when it comes time, we can edit them into slash commands.

ash663 commented on .gitignore:
```diff
@@ -151,6 +151,7 @@ crashlytics-build.properties
 
 
+config.json
 token.txt
```
We don't need a `token.txt` file anymore, right? Could we remove all references to it to avoid confusion?

reddit-tennis replied:
> We don't need a `token.txt` file anymore, right? Could we remove all references to it to avoid confusion?

I re-added it for historical purposes and so that people don't accidentally commit it again.

zork90 reviewed (approved):
Merge me daddy
